lib/schemaUtils.js

```diff
@@ -2707,7 +2707,7 @@ module.exports = {
       mismatches.push(mismatchObj);
     }
 
-    if (actualReqDesc !== expectedReqDesc) {
+    if ((!_.isEmpty(actualReqDesc) || !_.isEmpty(expectedReqDesc)) && (actualReqDesc !== expectedReqDesc)) {
```

Comment:
@VShingala too complicated. Add comments to describe the logic behind this check.